styles/style.css

```diff
@@ -0,0 +1,687 @@
+  }
+
+  .info__description-advantage {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: nowrap;
+    max-width: 100%;
+    margin-top: 67%;
+  }
+}
+
+@media screen and (max-width: 950px) {
+  .nav {
+    display: flex;
+    justify-content: space-between;
+    padding: 15px 30px;
+  }
+
+  .header {
+    margin-bottom: 35px;
+  }
+
+  .header-models {
+    display: none;
+  }
+
+  .first-floor__item {
+    background-position-y: 15%;
+  }
+
+  .stat {
+    width: 100%;
+    margin: 0 auto 50px;
+  }
+
+  .params {
+    display: flex;
+    flex-flow: column;
+    padding: 0 100px 0 150px;
+  }
+
+  .stat-power {
+    margin-right: 160px;
+  }
+
+  .wrap-params-one {
+    margin-bottom: 36px;
+  }
+
+  .stat__test-drive {
+    width: 140px;
+  }
+
+  .test-drive {
+    display: none;
+  }
+
+  .test-drive__red {
+    display: block;
+  }
+
+  .btn__first-floor {
+    background-color: #e51937;
+    color: #fff;
+  }
+}
+
+@media screen and (max-width: 770px) {
+  .nav {
+    display: flex;
+    justify-content: space-between;
+    padding: 15px 30px;
+    margin-bottom: 10px;
+  }
+
+  .header {
+    margin-bottom: 35px;
+  }
+
+  .header-models {
+    display: none;
+  }
+
+  .header-menu__item {
+    display: none;
+  }
+
+  .first-floor__item {
+    background-position-y: 19%;
+  }
+}
+
+@media screen and (max-width: 520px) {
+  .nav {
+    margin-bottom: 0;
+    padding: 15px 11px;
+  }
+
+  .title-text {
+    font-size: 60px;
+  }
+
+  .title-subtext {
+    font-size: 12px;
+  }
+
+  .stat {
+    width: auto;
+  }
+
+  .first-floor__item {
+    background-position-y: 15%;
+  }
+
+  .params {
+    width: 320px;
+    display: block;
+    padding: 0;
+  }
+
+  .stat__speed {
+    margin-right: 160px;
+  }
+
+  .speed {
+    width: 127px;
+    height: 53px;
+  }
+
+  .power-reserve {
+    width: 115px;
+    height: 53px;
+  }
+
+  .test-drive__red {
+    width: 135px;
+    height: 28px;
+  }
+
+  .stat__btn {
+    right: 80px;
+  }
+}
+
+@media screen and (max-width: 375px) {
+  .nav {
+    padding: 15px 5px;
+  }
+
+  .burger-btn {
+    margin-right: 10px;
+  }
+
+  .title-text {
+    font-size: 50px;
+  }
+
+  .title-subtext {
+    font-size: 10px;
+  }
+
+  .params {
+    margin-right: 10px;
+  }
+
+  .speed {
+    width: 125px;
+  }
+
+  .power-reserve {
+    width: 115px;
+  }
+
+  .stat__test-drive {
+    width: 0;
+  }
+
+  .test-drive__red {
+    margin-left: 10px;
+  }
+
+  .stat__btn {
+    left: 55px;
+  }
+}
+
+@media screen and (max-width: 270px) {
+  .first-floor__item {
+    background-position-y: 10%;
+  }
+
+  .params {
+    margin-left: 30px;
+  }
+
+  .wrap-params-one {
+    flex-flow: column;
+  }
+
+  .wrap-params-two {
+    flex-flow: column;
+    margin-right: 33px;
+  }
+
+  .stat__speed {
+    margin: 0 120px 50px 0;
+  }
+
+  .stat-power {
+    margin-right: 105px;
+  }
+
+  .stat__btn {
+    top: 50px;
+    left: 10px;
+  }
+
+  .swipe-down {
+    padding-top: 25px;
+  }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)